src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.test.tsx

```diff
@@ -45,7 +53,7 @@ const commonProps = {
 
 describe('<PopularSpeciesButton />', () => {
   beforeEach(() => {
-    jest.resetAllMocks();
+    jest.clearAllMocks();
```

Comment:
Curious as to why you switched this to `clearAllMocks`

Reply:
jest.resetAllMocks():
Resets all the mocks usage data, not their implementation. In other words, it only replaces fn.mock.calls and fn.mock.instances properties of a jest mock function.

jest.resetAllMocks()
A superset of clearAllMocks() which also takes care of resetting the implementation to a no return function. In other words, it will replace the mock function with a new jest.fn(), not just its fn.mock.calls and fn.mock.instances.

The tests were failing until I changed it but I guess by looking at their purpose above, I think `clearAllMocks` is what we should be using here?